examples/example-gateway/build/gen-code/endpoints/tchannel/baz/baz/simpleservice_compare.go

```diff
@@ -1,15 +1,15 @@
-// Code generated by thriftrw v1.16.1. DO NOT EDIT.
+// Code generated by thriftrw v1.18.0. DO NOT EDIT.
 // @generated
 
 package baz
 
 import (
-	"errors"
-	"fmt"
-	"go.uber.org/multierr"
-	"go.uber.org/thriftrw/wire"
-	"go.uber.org/zap/zapcore"
-	"strings"
+	errors "errors"
+	fmt "fmt"
+	multierr "go.uber.org/multierr"
+	wire "go.uber.org/thriftrw/wire"
+	zapcore "go.uber.org/zap/zapcore"
+	strings "strings"
```

Comment:
Why is thriftrw doing this?

Reply:
Refer thriftrw-go 1.17 [#405](https://github.com/thriftrw/thriftrw-go/pull/405)